### DISCUSSION TITLE
Is there a way to relegate threaded comments to a separate page, and just have an overview page?

### DISCUSSION THREAD

**Discussion by heaversm** [umputun/remark42#1668] [Ideas]
Would love to have a primary page that just shows all the comments, and a detail view that lists all the threads for a comment. Is this currently possible?

**Comment by umputun** [comment]
pls, use discussions for this type of ideas. The issues section is for the actual issues/bugs

moved

**Comment by umputun** [comment]
not sure what you mean exactly. Currently you can collapse and expand the comment tree. If you mean showing some popups as "Details" - no, this is not possible and not planned to be implemneted

**Comment by heaversm** [comment]
Collapsing the comment tree would become its own separate view: e.g:

<img width="353" alt="comment-list" src="https://github.com/umputun/remark42/assets/520541/a3e30424-7e73-4fb0-8baf-486fa63563de">
<img width="340" alt="comments-threaded" src="https://github.com/umputun/remark42/assets/520541/b10a0467-5ccf-4030-a381-a83275481db4">

**Reply by umputun** [reply]
This is up to your side. The count and the actual comments tree are two separate widgets. 

This is an example from one of the sites using remark42 in the similar way - https://blog.jetbrains.com/idea/

![image](https://github.com/umputun/remark42/assets/535880/507afd73-6831-42d7-8014-7843172a4248)

**Comment by heaversm** [comment]
Thanks - this is a helpful reference!
